test_requirements.txt

```diff
@@ -10,4 +10,5 @@ codecov
 asv
 gitpython
 pylint
-attrs>=19.2.0 # needed for hypothesis >=4.38.1 but not correctly honored by pip 19.2.3
\ No newline at end of file
+attrs>=19.2.0 # needed for hypothesis >=4.38.1 but not correctly honored by pip 19.2.3
+git+https://github.com/numpy/numpy-stubs.git@f3c6315738489983f5f37e1477ac68373d71b470 # no release yet so fix to a known working version
```

Comment:
just for my understanding, if this is not installed, then the type checking won't simply be as rigorous, right?

Reply:
Yes that is indeed the case. Basically if this is not installed all numpy types will be equivalent to Any